src/sentry/models/event.py

```diff
@@ -106,8 +112,9 @@ def get_event_metadata(self):
         etype = self.data.get('type', 'default')
         if 'metadata' not in self.data:
             # TODO(dcramer): remove after Dec 1 2016
```

Comment:
note for @mattrobenolt: the comment is a lie. Cannot be removed yet.

Reply:
Should I remove it? cc @dcramer - right now the handling of `event.message` is quite messy and hard to understand. @mitsuhiko and I were thinking about cleaning this up soon, but we'd need some input on which way to go.